### ISSUE TITLE
Future update to include newer watches?

### ISSUE THREAD

**Issue by Revisit3583** [sdgriggs/GarminDiscGolf#26]
I used this app with fenix 6 and it doesn't seem to be compatible at this time with forerunner 965, any chance there will be a future update to include newer watches?

**Comment by JoeMeister007** [comment]
Yes, it should be fairly simple to do so! No promise it'll happen ASAP, but I've been meaning to work on the app again

**Comment by Jvienne31** [comment]
Hello, 
I can't install your nice application because it doesn't seem to be compatible with the Venu 3.  Venu 2 plus and 3 are close. why is it not tagged Venu 3? do you have a way to install it on venu 3?

Thank you for your understanding.
